app/views/categories/_form.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_for([category, @lesson]) do |f| %>
+    <%= hidden_field_tag :current_user, current_user.id %>
+    <div class="btn-group pull-right">
+        <%= f.submit "Let's Start!", class:"btn btn-warning mt-4 btn-lg"%>
```

Comment:
There should be space present when you use RAILS code like this `<%= render 'new'  %>`
Yours is like this `<%= f.submit "Let's Start!", class:"btn btn-warning mt-4 btn-lg"%>`
Watch your spacing.